rules/windows/persistence_service_dll_unsigned.toml

```diff
@@ -0,0 +1,56 @@
+[metadata]
+creation_date = "2023/01/17"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
```

Comment:
What's the reason for min_stack : 8.4 ?

Reply:
`dll.Ext.relative_file_creation_time` is only available on Elastic Endpoint 8.4+

Reply:
can you change the ```min_stack_comments``` to reflect that?

Reply:
done